workshops/slack-pfp/README.md

```diff
@@ -0,0 +1,309 @@
+---
+name: Automating your Slack Profile Picture
+description: Make a program that changes your Slack profile picture based on the time of day.
+author: '@sampoder'
+image: 'https://cloud-45zk83i41.vercel.app/0screenshot_2020-11-22_at_3.27.11_pm.png'
+---
+
+# Automating your Slack Profile Picture
+
+<s>Having a plain old profile picture is so 2019. </s> Having a Slack Profile Picture that changes throughout the day is a great way to mix things up on Slack. In this workshop, we'll be learning how to make a dynamic profile picture that changes as the day progresses. We'll be using Node.js deployed as a [Vercel](https://vercel.com) Serverless Function and triggered by [cron-job.org](https://cron-job.org/en/). You'll be learning the basics of getting a Slack User Token, creating a Node.js program and deploying that program as a server less function.
+
+## Obtaining a Slack Token
+
+In my opinion this is the most painful part of this process.
+
+To begin you will need to be a member of a Slack Workspace. The easiest way to join one is to join [Hack Club's]([https://hackclub.com/slack/](https://hackclub.com/slack/)).
+
+Then visit the [Slack "Build" page]([https://api.slack.com/apps](https://api.slack.com/apps)). Click the large green button that says `Create New App`.
+
+Name it whatever you'd like, and choose the workspace you'd like the app to be for.
+
+On the next screen, find the `Add features and functionality` section and select `Permissions`.
+
+  
+![Slack API Configuration Screen with an arrow pointing to permissions](https://cloud-25utju6i9.vercel.app/0screenshot_2020-11-19_at_11.50.18_pm.png)
+
+  
+
+On this screen, scroll down to the `User Token Scopes` section and select the button that says `Add an OAuth Scope`.
+
+  
+
+![OAuth Scope Selection on Slack API page](https://cloud-1xkvojjl4.vercel.app/0screenshot_2020-11-19_at_11.55.09_pm.png)
+
+  
+
+In the pop-up box that follows, select `users.profile:write` and add that scope.
+
+  
+
+![OAuth Scope Selection on Slack API page completed](https://cloud-2ru7t5yz0.vercel.app/0screenshot_2020-11-20_at_12.00.03_am.png)
+
+  
+
+Scroll to the top of the page, click `Install to Workspace`, then `Allow` and then you will receive a token in return.
+
+  
+
+![Sample Token from Slack](https://cloud-1spwslhyd.vercel.app/0screenshot_2020-11-20_at_12.05.54_am.png)
+
+  
+
+Copy the generated token and store it in a safe space. Don't share it with anyone else (this is a fake token in the image, don't worry).
+
+  
+
+## Building our program
+
+To get started you are going to want to create a new Node.js  [repl.it](http://repl.it/)  project, [click here to do so]([https://repl.it/languages/nodejs](https://repl.it/languages/nodejs)).
```

Comment:
same link issue as above. see if you can fix it.

Reply:
I think I can